docs/resources/Channel.md

```diff
@@ -275,6 +276,31 @@ Represents a message sent in a channel within Discord.
 | SOURCE_MESSAGE_DELETED | 1 << 3 | the source message for this crosspost has been deleted (via Channel Following)    |
 | URGENT                 | 1 << 4 | this message came from the urgent message system                                  |
 
+
+###### Message Sticker Structure
+
+| Field            | Type      | Description                                                                           |
+| ---------------- | --------- | ------------------------------------------------------------------------------------- |
+| id               | snowflake | id of the sticker                                                                     |
+| pack_id          | snowflake | id of the pack the sticker is from                                                    |
+| name             | string    | name of the sticker                                                                   |
+| description      | string    | description of the sticker                                                            |
+| tags?            | string    | a comma-separated list of tags for the sticker                                        |
+| asset\*          | string    | sticker asset hash                                                                    |
+| preview_asset\*  | ?string   | sticker preview asset hash                                                            |
+| format_type      | integer   | [type of sticker format](#DOCS_RESOURCES_CHANNEL/message-object-sticker-format-types) |
+
+\* Sticker asset hashes currently cannot be resolved to an image.
```

Comment:
this doesn't seem true though? at least for png/apng stickers; https://cdn.discordapp.com/stickers/748293547572330607/067a41f0df36a1d0925078ded077824c.png?size=512 is the image from the `id` and `asset`

Reply:
@puddi Let's instead just say that the URL for fetching them is private

@advaith1 It's possible, but we don't want to yet document it. There's lots of undocumented but technically available things within our API. In this case, where we host stuff could be subject to change (since we have two URLs that resolve there right now, lol) and we don't want everyone scraping stickers yet outside of our experiment

Reply:
@msciotti comment updated!